README.md

```diff
@@ -28,7 +28,7 @@ $ npm install -g @google/repo
 You need to make your own [`config.yaml`](https://github.com/googleapis/github-repo-automation/blob/master/config.yaml.default) and put your GitHub token there. You can set the path to the config file with the `REPO_CONFIG_PATH` environment variable:
 
 ```sh
-$ cat /User/beckwith/.repo.yaml
+$ cat /User/beckwith/.config.yaml
```

Comment:
I think we default to `config.yaml` with no dot - maybe we shoudl do that here too?